.github/workflows/cd.yml

```diff
@@ -19,10 +19,15 @@ jobs:
     permissions:
       contents: read
       packages: write
-      # 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
```

Comment:
> buildx is a Docker CLI plugin for extended build capabilities with [BuildKit](https://github.com/moby/buildkit).

https://github.com/docker/buildx

We can't use the default Docker driver to build for multiple platforms, this plugin is needed